plugins/module_utils/module_index.py

```diff
@@ -156,6 +176,26 @@
                 },
             },
         },
+        "firewall_rules": {
+            "rules": "filter",
+            "php_requirements": [
+                "/usr/local/etc/inc/config.inc",
+                "/usr/local/etc/inc/util.inc",  # required for the service_log utility
+                "/usr/local/etc/inc/interfaces.inc",
+                "/usr/local/etc/inc/filter.inc",
+                "/usr/local/etc/inc/system.inc",
+            ],
+            "configure_functions": {
+                "system_cron_configure": {
+                    "name": "system_cron_configure",
+                    "configure_params": [],
```

Comment:
```suggestion
                    "configure_params": ["true"],
```
I would add `verbose=true` here which gives us more output if ansible was run with `-v`

-> https://github.com/opnsense/core/blob/24.1.5/src/etc/inc/system.inc#L1085